cmd/query/app/static_handler.go

```diff
@@ -68,15 +67,13 @@ type StaticAssetsHandlerOptions struct {
 
 // NewStaticAssetsHandler returns a StaticAssetsHandler
 func NewStaticAssetsHandler(staticAssetsRoot string, options StaticAssetsHandlerOptions) (*StaticAssetsHandler, error) {
-	if staticAssetsRoot == "" {
-		return nil, nil
-	}
-	if !strings.HasSuffix(staticAssetsRoot, "/") {
-		staticAssetsRoot = staticAssetsRoot + "/"
+	assetsFS, _ := fs.New()
+	if staticAssetsRoot != "" {
+		assetsFS = http.Dir(staticAssetsRoot)
 	}
-	indexBytes, err := ioutil.ReadFile(staticAssetsRoot + "index.html")
+	indexBytes, err := loadIndexHTML(assetsFS.Open)
 	if err != nil {
-		return nil, errors.Wrap(err, "Cannot read UI static assets")
+		return nil, errors.Wrap(err, "Cannot load index.html")
```

Comment:
loadIndexHTML seems to return errors that are sufficient, don't think the wrap is necessary.